### PR TITLE
Upgrade Stripe to support Bancontact

### PR DESCRIPTION
**Pull request by vstudenichnik-insoft**
https://app.asana.com/0/1204741807400778/1207362917086520/f

## Description

- Updated Bancontact Stripe integration to use Stripe Intent API

## Dependencies

- [swell-admin](https://gitlab.com/schema/swell-admin/-/merge_requests/2047)
- [schema-api](https://gitlab.com/schema/schema-api-server/-/merge_requests/1418)
- [swell-checkout](https://gitlab.com/schema/swell-checkout/-/merge_requests/240)
- [schema-vault](https://gitlab.com/schema/schema-vault/-/merge_requests/145)

## Testing

- Added unit tests to cover the creation/confirmation of a Stripe Setup Intent.

### CODE REVIEW

File: src/utils/stripe.js [hunk 1]
```diff
@@ -218,23 +180,21 @@ function getKlarnaConfirmationDetails(cart) {
...
  */
-async function createBancontactSource(stripe, cart) {
-  /** @type {CreateSourceData} */
-  const sourceObject = {
-    type: 'bancontact',
-    amount: Math.round(get(cart, 'grand_total', 0) * 100),
-    currency: toLower(get(cart, 'currency', 'eur')),
-    redirect: {
-      return_url: window.location.href,
+function getBancontactConfirmationDetails(cart) {
```
Comment (swellmike) [inline src/utils/stripe.js:186]: Add descriptive comments

File: dist/utils/index.mjs [hunk 2]
```diff
@@ -0,0 +1,6 @@
+import 'qs';
+export { D as base64Encode, e as camelCase, c as cloneDeep, d as defaultMethods, w as getLocationParams, j as getOptions, n as isEmpty, g as isEqual, k as isFunction, o as isLiveMode, b as isObject, i as isServer, l as loadScript, q as map, m as merge, p as pick, r as reduce, x as removeUrlParams, s as set, E as setOptions, f as snakeCase, C as stringifyQuery, a as toCamel, h as toCamelPath, F as toLower, t as toSnake, B as trimBoth, y as trimEnd, A as trimStart, u as uniq, v as vaultRequest } from '../index.738bc14a.mjs';
```
Comment (swellmike) [inline dist/utils/index.mjs:2]: The /dist files should not be part of this PR

File: src/utils/stripe.js (hunk 1 above)
Comment (vstudenichnik-insoft) [reply src/utils/stripe.js:188]: Done

File: dist/utils/index.mjs (hunk 2 above)
Comment (vstudenichnik-insoft) [reply dist/utils/index.mjs]: Done

**Review by awwit — APPROVED** [review]

**Review by swellmike — APPROVED** [review]
